azure/templates/deploy-spec.yml

```diff
@@ -16,7 +16,7 @@ parameters:
     type: boolean
 
 steps:
-- bash: 'sudo apt-get install npm curl jq -y'
+- bash: 'brew install npm curl'
```

Comment:
Why? We're not building this on MacOS

Reply:
We have some dependancy issues: https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId=5861&view=logs&j=5a36e16c-a3c2-54ba-30ad-ce549a67a4f8&t=78240962-52ad-5088-1b61-69ef3ccb45e9

Also we use brew to install terraform

Reply:
We shouldn't be installing terraform with brew either really